mirage/lE.ml

```diff
@@ -0,0 +1,179 @@
+(* (c) Hannes Menhert *)
```

Comment:
```suggestion
```

there's no other file that claims any specific copyright, let's just leave that out...